python/trans_to_AA.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jul 18 12:28:33 2018
+
+@author: jordanzehr
+"""
+
+import argparse
+from Bio import SeqIO
+from Bio.Seq import Seq
+
+parser = argparse.ArgumentParser(
+    description='Create JSON to be consumed by dashboard for final output.'
+)
+
+parser.add_argument(
+    '-p', '--patient',
+    metavar='PATIENT',
+    type=str,
+    help='Patient ID.',
+    required=True
+)
+
+parser.add_argument(
+    '-g', '--gene',
+    metavar='GENE',
+    type=str,
+    help='Which V-gene to build (integer from 1 to 7).',
+    required=True
+)
+
+args = parser.parse_args()
+patient_id = args.patient
+v_gene = args.gene
+
+nuc = 'data/out/%s/V%s_unaligned.fasta' % (patient_id, v_gene)
+
+
+records = list(SeqIO.parse(nuc, 'fasta'))
+k= []
+j = []
+q = []
+r = []
+seqs_with_stops = []
+for seq_record in records:
+    x = len(seq_record)%3.0
+    y = int(len(seq_record.translate()))
```

Comment:
I believe you want `seq_record.seq.translate()` instead of `seq_record.translate()`.